### PR TITLE
Scenario refactor. Account page added.

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/main/java/ninjaStore/ui/pages/AccountPage.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package ninjaStore.ui.pages;
```
Comment (titor-oopart) [inline src/main/java/ninjaStore/ui/pages/AccountPage.java:12]: add documentation

File: src/test/resources/features/buy.feature [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Feature: Buy products
+  Scenario: Add to cart from home page
+    Given the user goes to home page
+    When adds MacBook to cart
+    Then success message is shown
+
+  Scenario: Checkout from shopping cart
+    Given the user goes to shopping cart page
+    When the user presses checkout button
+    Then billing details form is shown
```
Comment (titor-oopart) [inline src/test/resources/features/buy.feature]: add empty line

File: src/test/resources/features/login.feature [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+    When the user login entering his email and password
+    Then "My Account" tittle is shown
+
+  @Login
+  Scenario: Login The Ninja Store page with an existing account
+    Given the user goes to login page
+    When the user login entering his email and password
+    Then "My Account" tittle is shown
+
+
```
Comment (titor-oopart) [inline src/test/resources/features/login.feature]: delete unnecessary lines

File: config/checkstyle/checkstyle.xml [hunk 4]
```diff
@@ -87,7 +87,7 @@
         <module name="LocalVariableName"/>
         <module name="MemberName"/>
         <module name="MethodName"/>
-        <module name="PackageName"/>
+<!--        <module name="PackageName"/>-->
```
Comment (titor-oopart) [inline config/checkstyle/checkstyle.xml]: aline with others modules

File: src/main/java/ninjaStore/ui/pages/AccountPage.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/ninjaStore/ui/pages/AccountPage.java:12]: done

File: src/test/resources/features/login.feature (hunk 3 above)
Comment (MelissaRoman06) [reply src/test/resources/features/login.feature]: done

File: config/checkstyle/checkstyle.xml (hunk 4 above)
Comment (MelissaRoman06) [reply config/checkstyle/checkstyle.xml]: done

**Pull request merged**